karapace/kafka_rest_apis/consumer_manager.py

```diff
@@ -155,12 +155,12 @@ def _update_partition_assignments(consumer: KafkaConsumer):
     # CONSUMER
     async def create_consumer(self, group_name: str, request_data: dict, content_type: str):
         group_name = group_name.strip("/")
-        self.log.info("Create consumer request for group  %s", group_name)
+        self._log.info("Create consumer request for group  %s", group_name)
```

Comment:
I would combine the return value of `new_name` to the log. Also consider if this is debug log instead of info.
The group and name is logged as info also on the line 173.

Reply:
done